test/dependencies/iptb/main.go

```diff
@@ -23,6 +23,8 @@ import (
 	manet "github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/jbenet/go-multiaddr-net"
 )
 
+var setupOpt = func(cmd *exec.Cmd) {}
```

Comment:
https://github.com/whyrusleeping/iptb/commit/b21f95b87a2275911fda072ee916dd2da72b436a is the commit I think